jetstream/defaults/firefox_desktop.toml

```diff
@@ -12,6 +12,8 @@ weekly = [
   "search_count",
   "qualified_cumulative_days_of_use",
   "is_default_browser",
+  "sponsored_pocket_ctr",
+  "sponsored_tile_ctr",
   "client_level_daily_active_users_v2",
 ]
 
```

Comment:
These metrics need a statistic associated with them. Historically that's been defined in the (confusingly named) `/jetstream/definitions/firefox_desktop.toml` [file](https://github.com/mozilla/metric-hub/blob/main/jetstream/definitions/firefox_desktop.toml)

Reply:
Thank you I was just trying to figure out what was going on there